content/agent/faq/agent-configuration-file-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Agent configuration file migration
+kind: faq
+further_reading:
+---
+
+[If you haven't done it already, upgrade your Agent v5 to Agent v6][1]
+
+To automatically transition Agent configuration paths and formats from Agent v5 to Agent v6, use the `import` command. The command parses an existing v5 `datadog.conf` and converts the configuration options to the new v6 `datadog.yaml` format. It also copies configuration files for checks that are currently enabled.
+
+{{< tabs >}}
+{{% tab "Linux" %}}
+
+`sudo -u dd-agent -- datadog-agent import`
+
+{{% /tab %}}
+{{% tab "macOS" %}}
+
+`datadog-agent import <OLD_CONFIGURATION_DIRECTORY> <DESTINATION_DIRECTORY>`
+
+With:
+
+* `<OLD_CONFIGURATION_DIRECTORY>` is the directory containing the `datadog.conf` file
```

Comment:
let's mention the default users should use: `/opt/datadog-agent/etc/`

Reply:
For MacOS it's not `~/.datadog-agent/` ?

Reply:
it is, _but_ the files in `~/.datadog-agent/` are symlinks to `/opt/datadog-agent/etc/`, so for the purpose of the `import` command it's better to use `/opt/datadog-agent/etc`.

In any case, users shouldn't have to run the `import` command manually on Mac since there's only a `one step install` on macOS, and it runs the `import` command on its own